b2/raw_api.py

```diff
@@ -570,27 +619,27 @@ def test_raw_api_helper(raw_api):
     print('b2_download_file_by_id (auth)')
     download_dest = DownloadDestBytes()
     raw_api.download_file_by_id(download_url, account_auth_token, file_id, download_dest)
-    assert file_contents == download_dest.bytes_io.getvalue()
+    assert file_contents == download_dest.get_bytes_written()
```

Comment:
I don't know if you'll want this change or not.  Without it, pre-commit.sh was failing here for me, but I don't recall failing integration tests in Travis before, so maybe something is different for my environment.  Either way, it kind of looks like the `DownloadDestBytes` author intended for `get_bytes_written()` to be used.